XRTK-Core/Packages/com.xrtk.core/Runtime/Providers/Controllers/OpenVR/OculusTouchOpenVRController.cs

```diff
@@ -63,7 +65,34 @@ public OculusTouchOpenVRController(IMixedRealityControllerDataProvider controlle
             new MixedRealityInteractionMapping("Button.One Touch", AxisType.Digital, DeviceInputType.ButtonPress, KeyCode.JoystickButton10),
             new MixedRealityInteractionMapping("Button.Two Touch", AxisType.Digital, DeviceInputType.ButtonPress, KeyCode.JoystickButton11),
             new MixedRealityInteractionMapping("Touch.SecondaryThumbRest Touch", AxisType.Digital, DeviceInputType.ThumbTouch, KeyCode.JoystickButton19),
-            new MixedRealityInteractionMapping("Touch.SecondaryThumbRest Near Touch", AxisType.Digital, DeviceInputType.ThumbNearTouch, ControllerMappingLibrary.AXIS_18)
+            new MixedRealityInteractionMapping("Touch.SecondaryThumbRest Near Touch", AxisType.Digital, DeviceInputType.ThumbNearTouch, ControllerMappingLibrary.AXIS_18),
+            new MixedRealityInteractionMapping("Grip Pose", AxisType.SixDof, DeviceInputType.SpatialGrip)
         };
+
+        public override void UpdateController()
+        {
+            base.UpdateController();
+
+            if (TrackingState == TrackingState.Tracked)
+            {
+                for (int i = 0; i < Interactions?.Length; i++)
+                {
+                    var interactionMapping = Interactions[i];
+                    switch (interactionMapping.InputType)
+                    {
+                        case DeviceInputType.SpatialGrip:
+                            UpdateSpatialGripData(interactionMapping);
+                            interactionMapping.RaiseInputAction(InputSource, ControllerHandedness);
+                            break;
+                    }
+                }
+            }
+        }
+
+        private void UpdateSpatialGripData(MixedRealityInteractionMapping interactionMapping)
+        {
+            Debug.Assert(interactionMapping.AxisType == AxisType.SixDof);
+            interactionMapping.PoseData = CurrentControllerPose;
```

Comment:
See comment on Oculus PR, needs GripPoseOffset update.

Should also look to adding the GripPoseOffset variable to the BaseController definition for use in all controllers (but only implement in those that need it for now)